src/server/utils/onModuleLoad.js

```diff
@@ -108,9 +121,7 @@ export default function onModuleLoad({
   }
 
   if (moduleName === serverStateConfig.rootModuleName) {
-    if (!csp) {
-      throw new Error('Root module must provide a valid content security policy');
-    }
+    cspCheck(csp);
```

Comment:
I don't love this floating function call. Maybe it should return true/false, and we keep the original if check ? Just personal preference, feel free to disagree.

Reply:
I actually kind of agree, functionally it would be the same, But leaving this as is, is not the cleanliest. But what I will say is that it reduces a spot for complexity that could potentially be used later?

Reply:
I could go either way, but in either case this function needs a better name. Something along the lines of `validateCspIsPresent` if throwing here, and `isCspPresent` if returning a bool